### PR TITLE
Fixed #33893 -- Reverted "Fixed #28889 -- Prevented double submission of admin forms."

### PR DESCRIPTION
**Pull request by fabtjar**
As per ticket [#33893](https://code.djangoproject.com/ticket/33893): Revert prevented double submission of admin form which cause a regression from ticket [#28889](https://code.djangoproject.com/ticket/28889).

### CODE REVIEW

File: docs/releases/4.1.1.txt [hunk 1]
```diff
@@ -18,3 +18,6 @@ Bugfixes
 * Fixed a regression in Django 4.1 that caused a crash of the ``test``
   management command when running in parallel and ``multiprocessing`` start
   method is ``spawn`` (:ticket:`33891`).
+
+* Reverted a fix (:ticket:`28889`) to prevented double submission of admin
+  forms which caused a regression (:ticket:`33893`).
```
Comment (felixxm) [inline docs/releases/4.1.1.txt:23]: Maybe:
```
* Fixed a regression in Django 4.1 that caused an incorrect redirection to the
  admin changelist view when using "Save and continue editing" and "Save and
  add another" options (:ticket:`33893`).
```
It should be clear for users what are has been fixed.

**Review by felixxm — COMMENTED** [review]
@fabtjar Thanks :+1: Can we add a regression test?

**Comment by felixxm** [comment]
> @fabtjar Thanks +1 Can we add a regression test?

For example:
```diff
diff --git a/tests/admin_views/tests.py b/tests/admin_views/tests.py
index 88468e883c..632a49897b 100644
--- a/tests/admin_views/tests.py
+++ b/tests/admin_views/tests.py
@@ -6432,6 +6432,23 @@ class SeleniumTests(AdminSeleniumTestCase):
         self.assertEqual(traveler.living_country.name, "Italy")
         self.assertEqual(traveler.favorite_country_to_vacation.name, "Qatar")
 
+    def test_redirect_on_add_view_add_another_button(self):
+        from selenium.webdriver.common.by import By
+
+        self.admin_login(
+            username="super", password="secret", login_url=reverse("admin:index")
+        )
+        add_url = reverse("admin7:admin_views_section_add")
+        self.selenium.get(self.live_server_url + add_url)
+        name_input = self.selenium.find_element(By.ID, "id_name")
+        name_input.send_keys("Test section 1")
+        self.selenium.find_element(By.XPATH, '//input[@value="Save and add another"]').click()
+        self.assertEqual(Section.objects.count(), 1)
+        name_input = self.selenium.find_element(By.ID, "id_name")
+        name_input.send_keys("Test section 2")
+        self.selenium.find_element(By.XPATH, '//input[@value="Save and add another"]').click()
+        self.assertEqual(Section.objects.count(), 2)
+
 
 @override_settings(ROOT_URLCONF="admin_views.urls")
 class ReadonlyTest(AdminFieldExtractionMixin, TestCase):

```

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Comment by felixxm** [comment]
@fabtjar Thanks :+1: Welcome aboard :boat:

**Comment by fabtjar** [comment]
@felixxm Thanks for the help 👍

**Pull request merged**